### PR TITLE
Make installable

### PR DESCRIPTION
**Pull request by c-dinkel**
This makes the app installable by providing a [manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest) and [service worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API).

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "short_name": "OpenSCD",
+  "name": "OpenSCD",
+  "description": "A substation configuration description editor using SCL IEC 61850-6",
```
Comment (JakobVogelsang) [inline manifest.json:4]: Das müssen wir leider umschreiben aus verschiedenen Gründen.

"An editor for XML files as described by IEC 61850-6"

File: src/open-scd.ts [hunk 2]
```diff
@@ -299,11 +315,28 @@ export class OpenSCD extends Setting(
         </mwc-top-app-bar-fixed>
       </mwc-drawer>
 
-      ${cache(
-        until(
-          this.plugins.editors[this.activeTab].getContent(),
-          html`<mwc-linear-progress indeterminate></mwc-linear-progress>`
-        )
```
Comment (JakobVogelsang) [inline src/open-scd.ts:288]: Why did you delete this?

File: manifest.json (hunk 1 above)
Comment (JakobVogelsang) [inline manifest.json:4]: Das müssen wir leider umschreiben aus verschiedenen Gründen.

"An editor for XML files as described by IEC 61850-6"

**Review by JakobVogelsang — CHANGES_REQUESTED** [review]
One minor change

File: manifest.json (hunk 1 above)
Comment (c-dinkel) [reply manifest.json]: Done.
